docs/user/dashboard/vega-reference.asciidoc

```diff
@@ -250,8 +250,9 @@ configuration:
       "longitude": -74,   // default 0
       "zoom": 7,          // default 2
 
-      // defaults to "default". Use false to disable base layer.
-      "mapStyle": false,
+      // This configures the default tile layer of Maps.
```

Comment:
There is enough complexity with `mapStyle` that it should not be explained in a comment. How about moving this into its own paragraph after the example. Something like the below

Use `mapStyle` to configure a base map layer for your vega map. `mapStyle` can be set to one of the following:
* Set to `false` to disable base layer.
* Set to `"default"` to use Kibana's default base map layer.
* https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts tile layers. Set `mapStyle` to any EMS tile layer id. 
* Set to `"map.tilemap.url"` to use tile map service configured in kibana.yml. See map.tilemap.url in <<settings>> for details.